cmd/kube-proxy/app/server.go

```diff
@@ -68,7 +72,7 @@ type ProxyServer struct {
 
 // NewProxyServer creates a new ProxyServer object with default parameters
 func NewProxyServer() *ProxyServer {
-	return &ProxyServer{
+	s := ProxyServer{
```

Comment:
not really on purpose. I originally killed this func, clayton complained/said they needed it, so I restored it. But I restored it by hand, following the same convention as the other New*Server() functions. So it isn't needed, but does make the code look slightly more like the other similar code...